packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -726,6 +726,13 @@ export function attach(
         'color: purple;',
         'color: black;',
       );
+      console.log(
+        new Error().stack
+          .split('\n')
+          .slice(1)
+          .join('\n'),
+      );
```

Comment:
This isn't necessary as part of the PR but I found it helpful when debugging things.